src/utils/format.js

```diff
@@ -163,3 +184,37 @@ export function formatFiatChangeExtended(change: ?BigNumber, initialBalance: ?Bi
 
   return null;
 }
+
+/**
+ * Format token value change without + sign.
+ * Default number of decimal places depends on token symbol and trailing zeroes are stripped.
+ */
+export function formatTokenValue(value: ?BigNumber, symbol?: string, options?: FormatValueOptions) {
+  if (!value || !value.isFinite()) return null;
+
+  const decimalPlaces = getDecimalPlaces(symbol);
+  const formattedValue = formatValue(value.abs(), { decimalPlaces, ...options });
+
+  return value.gte(0)
+    ? t('tokenValue', { value: formattedValue, token: symbol })
+    : t('negativeTokenValue', { value: formattedValue, token: symbol });
+}
+
+/**
+ * Format token value change with +/- sign.
+ * Default number of decimal places depends on token symbol and trailing zeroes are stripped.
+ */
+export function formatTokenChange(value: ?BigNumber, symbol?: string, options?: FormatValueOptions) {
+  if (!value || !value.isFinite()) return null;
+
+  const decimalPlaces = getDecimalPlaces(symbol);
+  const formattedValue = formatValue(value.abs(), { decimalPlaces, ...options });
+
+  return value.gte(0)
+    ? t('positiveTokenValue', { value: formattedValue, token: symbol })
+    : t('negativeTokenValue', { value: formattedValue, token: symbol });
+}
+
+export function formatHexAddress(address: string) {
```

Comment:
There's `humanizeHexString` on `utils/common` and `elipsizeAddress` on `utils/feedData`, maybe we should either get rid of others and replace with single method or mark those as deprecated, wdyt?